modules/stanford_soe_helper_news/stanford_soe_helper_news.install

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * @file
+ */
+
+
+/**
+ * Implements hook_install().
+ */
+
+function stanford_soe_helper_news_install() {
+
+  // Create a bean link block
+  // See https://www.drupal.org/node/1436186
+  // and http://api.drupalecommerce.org/api/misc/bean%21bean.test/function/BeanTests%3A%3AtestBeanAPI/7.x
+  
+  $text =  l(t('See more news'), '/news/recent-news',
```

Comment:
It would be much more robust if this were not a stanford_large_block but rather a custom block or a view. The link `/news/recent-news` breaks on all of the development sites as they are subdirectory rather than subdomain. Please change this to a custom block that uses the l() or url() function to generate the link dynamically each time rather than saving it to the database. This is bad for portability. This will also save you on checking for wether or not the block exists and having to write drush functions to remove them. 

Please re-write this in hook_block_info and remove the hook_requirement and drush plugin.